docs/src/comp-kubeelasti-vs-knative.md

```
@@ -0,0 +1,251 @@
+metadata:
+  name: example-trigger
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: com.example.event
+  subscriber:
+    ref:
+      kind: Service
+      name: example-service
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)